pkgs/development/python-modules/moderngl_window/default.nix

```diff
@@ -1,4 +1,5 @@
-{ lib
+{ stdenv
```

Comment:
I don't think this makes a difference. We will only be using the `isDarwin` attribute and will not fetch a C compiler anyway. I think it would be better to use `stdenv` for consistency in case of overrides and things like that.